go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/strangelove-ventures/async-icq/v4 v4.0.0-rc0
 	github.com/strangelove-ventures/packet-forward-middleware/v4 v4.0.4
 	github.com/stretchr/testify v1.8.1
-	github.com/tendermint/tendermint v0.34.24
+	github.com/tendermint/tendermint v0.34.25
```

Comment:
doesn't this also need to change in the replace?

```
github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.24 // replace with  c0.34.25
```

Reply:
Also noticed that CI is not passing for that tag, but it seems to be broken on most tags/branches too

Reply:
yes, thanks for pointing out! Seems that this is because of the tedermint version mismatches between our cosmos-sdk and Osmosis repo, investigating this further